internal/meta/data.go

```diff
@@ -14,61 +14,62 @@ const (
+	Frames              int           `meta:"FrameCount,AnimationFrames"`
+	Codec               string        `meta:"CompressorID,VideoCodecID,CodecID,OtherFormat,FileType"`
+	Title               string        `meta:"Title,Headline" xmp:"dc:title" dc:"title,title.Alt"`
+	Description         string        `meta:"Description,ImageDescription,Caption,Caption-Abstract" xmp:"Description,Description.Alt"`
+	Subject             string        `meta:"Subject,PersonInImage,ObjectName,HierarchicalSubject,CatalogSets" xmp:"Subject"`
+	Keywords            Keywords      `meta:"Keywords"`
+	Favorite            bool          `meta:"Favorite"`
+	Notes               string        `meta:"Comment,UserComment"`
+	Artist              string        `meta:"Artist,Creator,By-line,OwnerName,Owner" xmp:"Creator"`
+	Copyright           string        `meta:"Rights,Copyright,CopyrightNotice,WebStatement" xmp:"Rights,Rights.Alt"`
+	License             string        `meta:"UsageTerms,License"`
+	Projection          string        `meta:"ProjectionType"`
+	ColorProfile        string        `meta:"ICCProfileName,ProfileDescription"`
+	CameraMake          string        `meta:"CameraMake,Make" xmp:"Make"`
+	CameraModel         string        `meta:"CameraModel,Model" xmp:"Model"`
+	CameraOwner         string        `meta:"OwnerName"`
+	CameraSerial        string        `meta:"SerialNumber"`
+	LensMake            string        `meta:"LensMake"`
+	LensModel           string        `meta:"Lens,LensModel" xmp:"LensModel"`
+	Software            string        `meta:"Software,CreatorTool,HistorySoftwareAgent,ProcessingSoftware"`
+	Flash               bool          `meta:"FlashFired"`
+	FocalLength         int           `meta:"FocalLength,FocalLengthIn35mmFormat"`
+	FocalDistance       float64       `meta:"HyperfocalDistance"`
+	Exposure            string        `meta:"ExposureTime,ShutterSpeedValue,ShutterSpeed,TargetExposureTime"`
+	Aperture            float32       `meta:"ApertureValue,Aperture"`
+	FNumber             float32       `meta:"FNumber"`
+	Iso                 int           `meta:"ISO"`
+	ImageType           int           `meta:"HDRImageType"`
+	GPSPosition         string        `meta:"GPSPosition"`
+	GPSLatitude         string        `meta:"GPSLatitude"`
+	GPSLongitude        string        `meta:"GPSLongitude"`
+	Lat                 float32       `meta:"-"`
+	Lng                 float32       `meta:"-"`
+	Altitude            float64       `meta:"GlobalAltitude,GPSAltitude"`
+	Width               int           `meta:"ImageWidth,PixelXDimension,ExifImageWidth,SourceImageWidth"`
+	Height              int           `meta:"ImageHeight,ImageLength,PixelYDimension,ExifImageHeight,SourceImageHeight"`
+	Orientation         int           `meta:"-"`
+	Rotation            int           `meta:"Rotation"`
+	Views               int           `meta:"-"`
+	Albums              []string      `meta:"-"`
+	EmbeddedVideo       string        `meta:"-"`
```

Comment:
Not directly related to this PR, but just to share, in my PR I also changed this field to boolean: https://github.com/photoprism/photoprism/pull/3709/files#diff-5e54ffefc2cd1c77d57258d92f2c45a9d9cebf1ad8b70519309cd32eae287de1R370 thinking that we don't actually need the field name, we just need to know if the video exists, as it is used here: https://github.com/photoprism/photoprism/pull/3709/files#diff-b92b8f546ec67d3a9b9e70bb31cd50416ed9489173fa35e7687aa123fd13e7c9R60, this is the case for both the MP4 header approach and the `DirectoryItemLength` approach I guess so we could keep it in mind